packages/full-self-signing/eslint.config.cjs

```diff
@@ -9,6 +9,7 @@ module.exports = [
         'error',
         {
           ignoredFiles: ['{projectRoot}/eslint.config.{js,cjs,mjs}'],
+          ignoredDependencies: ['bs58'],
```

Comment:
fyi @spacesailor24 eslint was complaining that this dependency is unused but it is definitely used.  i couldn't figure out the underlying issue so i just ignored it from the eslint config.